client/src/pages/Login.js

```diff
@@ -1,9 +1,112 @@
 import React, { Component } from "react";
+import { withStyles } from "@material-ui/core/styles";
+
+import NavBar from "../components/NavBar";
+import TextField from "@material-ui/core/TextField";
+import Button from "@material-ui/core/Button";
+import Grid from "@material-ui/core/Grid";
+
+const loginPageStyle = theme => ({
+  container: {
+    display: "flex",
+    flexWrap: "wrap"
+  },
+  textField: {
+    marginLeft: theme.spacing(1),
+    marginRight: theme.spacing(1)
+  },
+  button: {
+    color: "white",
+    marginTop: theme.spacing(3),
+    marginBottom: theme.spacing(1)
+  }
+});
 
 class Login extends Component {
+  state = {
+    userEmail: "",
+    userPassword: "",
+    errorMessage: ""
+  };
+  handleSubmit = async e => {
+    e.preventDefault();
+    //direct user to profile page
```

Comment:
So it doesn't actually logs in yet? If that's because the server code for this isn't merged yet, you can just assume it exists and implement the client side.

Reply:
Yeah it doesn't technically log in yet, but @alaindavis90 has implemented that section in his branch feature-auth-server because he has a better understanding of jwt and the right GET request to make.

Reply:
Got it. Keep in mind that you guys can always work on each other's branch when you need help with a feature. It's better than mixing code for different issues on the same PR and it keeps the merges clean.

Reply:
We'll try to do that in future.